Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageTest.cs

```diff
@@ -24,7 +24,7 @@ public void InitializeStorageTest()
         [TestMethod]
         public void CountEmptyStorage()
         {
-            var count = _storage.CountLogsAsync(StorageTestChannelName).RunNotAsync();
+            var count = _storage.CountLogsAsync(StorageTestChannelName).GetAwaiter().GetResult();
```

Comment:
Tests can be `async` and I've already replaced it partially.
`async`/`await` looks like right replacement instead of `GetAwaiter().GetResult()`
You don't necessarily change all the tests in the project, only those that relate to your fix.